UI/window-basic-main.cpp

```diff
@@ -4867,6 +4868,8 @@ void OBSBasic::ReplayBufferStart()
 	if (!outputHandler || !outputHandler->replayBuffer)
 		return;
 
+	replayBufferButton->setChecked(true);
+
```

Comment:
I'd personally remove this blank line to group it with the `replayBufferButton->setText()` below.